deb-get

```diff
@@ -2595,6 +2596,12 @@ if [ -e /etc/os-release ]; then
         debian) UPSTREAM_CODENAME=$(grep VERSION_CODENAME /etc/os-release | cut -d'=' -f2);;
         *) UPSTREAM_CODENAME="";;
     esac
+
+    # Set UPSTREAM_CODENAME to focal OS_ID_PRETTY is Kali GNU/Linux Rolling
+    # To emulate a Debian derived release
+    if [ "${OS_ID_PRETTY}" == "Kali GNU/Linux Rolling" ]; then
```

Comment:
This is not how UPSTREAM_CODENAME should be handled. If the current solution for detecting UPSTREAM_CODENAME does not work for Kali, then a better way should be proposed.